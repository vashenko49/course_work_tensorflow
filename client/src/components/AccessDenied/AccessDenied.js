import React, { Component } from 'react';
import Container from '@material-ui/core/Container';
import { Typography } from '@material-ui/core';

class AccessDenied extends Component {
  render() {
    return (
      <Container>
        <Typography variant={'h2'}>
          You do not have enough privileges to access this service.
        </Typography>
      </Container>
    );
  }
}

export default AccessDenied;
