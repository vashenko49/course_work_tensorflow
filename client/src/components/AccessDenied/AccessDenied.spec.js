import React from 'react';
import { shallow } from 'enzyme';
import AccessDenied from './AccessDenied';

describe('AccessDenied', () => {
  describe('AccessDenied initial', () => {
    const accessDenied = shallow(<AccessDenied {...{}} />);
    it('renders properly', () => {
      expect(accessDenied).toMatchSnapshot();
    });
  });
});
