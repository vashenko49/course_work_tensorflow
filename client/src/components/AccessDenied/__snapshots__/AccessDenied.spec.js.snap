// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccessDenied AccessDenied initial renders properly 1`] = `
<WithStyles(ForwardRef(Container))>
  <WithStyles(ForwardRef(Typography))
    variant="h2"
  >
    You do not have enough privileges to access this service.
  </WithStyles(ForwardRef(Typography))>
</WithStyles(ForwardRef(Container))>
`;
